### PR TITLE
Add markdown tutorial file

### PR DESCRIPTION
**Pull request by jamesadevine**
This is a PR for the page we'd like to use for the Jacdac user study.

### CODE REVIEW

**Comment by pelikhan** [comment]
Could you crop all the margins around gifs to try to keep their sizes down?

Also, use minimal mode when capturing something like bing, no political news in bing.

File: src/pages/clients/makecode/tutorials/study.mdx [hunk 1]
```diff
@@ -0,0 +1,153 @@
...
+order: 0
+---
+
+## Before you code...
+
+The **[Out of the box program](/hardware/kit/oob/)** is a Jacdac program
+that automatically reacts to Jacdac modules connected to the micro:bit v2. The micro:bit you were sent should already be loaded with this program.
+Before getting into coding Jacdac programs, try plugging Jacdac modules into your micro:bit and see what happens!
+
+![Animation of a button connected to the micro:bit V2](../../../hardware/kit/oob.gif)
```
Comment (pelikhan) [inline src/pages/clients/makecode/tutorials/study.mdx:12]: Use absolutely oaths

Comment (pelikhan) [reply src/pages/clients/makecode/tutorials/study.mdx:12]: paths

**Comment by jamesadevine** [comment]
Yup, can do! I didn't realise there was such a mode for bing

**Review by pelikhan — APPROVED** [review]

**Comment by pelikhan** [comment]
We could also launch a step by step tutorial in makecode itself.

File: src/pages/clients/makecode/tutorials/study.mdx (hunk 1 above)
Comment (jamesadevine) [reply src/pages/clients/makecode/tutorials/study.mdx:12]: Could you give an example of the absolute path? I'm not sure which folder the files will be referenced from if that makes sense.

**Pull request merged**